### PR TITLE
New logger

### PR DESCRIPTION
**Pull request by ilyastrodubtsev**
The old logger has been replaced with a more informative one.
Now we display:
* Time
* Error level
* The name of the file in which the event occurred
* Function name
* Path to file with line number

### CODE REVIEW

File: gatewayconfig/logger.py [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
-# create logger
-logger = logging.getLogger('gatewayconfig')
-logger.setLevel(LOGLEVEL)
 
-# create console handler and set level to debug
-stream_handler = logging.StreamHandler()
-stream_handler.setLevel(LOGLEVEL)
+def get_stream_handler():
+    stream_handler = logging.StreamHandler()
+    stream_handler.setLevel(logging.DEBUG)
```
Comment (marvinmarnold) [inline gatewayconfig/logger.py:8]: Lets continue to be able to overwrite with an environment variable `os.environ.get("LOGLEVEL", "DEBUG")`

File: gatewayconfig/logger.py [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
 
-# create formatter
-formatter = logging.Formatter("%(asctime)s;%(levelname)s;%(message)s",
-                              "%Y-%m-%d %H:%M:%S")
 
-# add formatter to stream_handler
-stream_handler.setFormatter(formatter)
+def get_logger(name):
+    logger = logging.getLogger(name)
+    logger.setLevel(logging.DEBUG)
```
Comment (marvinmarnold) [inline gatewayconfig/logger.py:15]: Env variable here too

**Review by marvinmarnold — CHANGES_REQUESTED** [review]
Looks great. Just the one comment about using optional env variable for loglevel. Don't forget to squash into a single commit also.

File: gatewayconfig/logger.py (hunk 1 above)
Comment (ilyastrodubtsev) [reply gatewayconfig/logger.py]: Done. https://github.com/NebraLtd/hm-config/blob/ca7b5e3e05041b9fdaaa238e3240c1f1e38f1cfd/gatewayconfig/logger.py#L10

File: gatewayconfig/logger.py (hunk 2 above)
Comment (ilyastrodubtsev) [reply gatewayconfig/logger.py]: Done. https://github.com/NebraLtd/hm-config/blob/ca7b5e3e05041b9fdaaa238e3240c1f1e38f1cfd/gatewayconfig/logger.py#L17

**Review by marvinmarnold — APPROVED** [review]

**Pull request merged**